configure.ac

```diff
@@ -18,8 +18,11 @@ PKG_CHECK_MODULES([lzma], [liblzma])
 PKG_CHECK_MODULES([zlib], [zlib])
 AC_ARG_ENABLE(
 	[bzip2],
-	AS_HELP_STRING([--disable-bzip2],[Do not use bzip2 compression (uses bzip2 by default)])
+	AS_HELP_STRING([--disable-bzip2],[Do not use bzip2 compression (uses bzip2 by default)]),
+	BZIP='no',
+	BZIP='yes'
```

Comment:
@bradTpeters Can you make the same change for the AC_ARG_ENABLE uses you modified (i.e. set the shell variables within the appropriate AS_IF instead)?